lib/gui/components/svg-icon/styles/_svg-icon.scss

```diff
@@ -0,0 +1,13 @@
+
+svg {
+  width: 100%;
+  height: 100%;
+}
+
+svg-icon {
+  display: inline-block;
+
+  > div[disabled] path {
```

Comment:
Can we move this particular rule to the `main.scss` file? This component doesn't control what its non-disabled color is, so I don't think it knows enough about himself to pick a proper disabled color.